validating-admission-policy-template/template.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: [[forms[0].name]]
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+      - apiGroups: [ "[[forms[0].apiGroups]]" ]
+        apiVersions: [ "[[forms[0].apiVersions]]" ]
+        operations: [ [[forms[0].operations ]] ]
+        resources: [ [[forms[0].resources]] ]
```

Comment:
are quotes missing on `resources` and `operations` ? or maybe they should be removed from `apiGroups` and `apiVersions`?